src/pages/signin/SignInPageLayout/SignInPageLayoutWide.js

```diff
@@ -35,14 +45,23 @@ const SignInPageLayoutWide = ({children}) => (
                         Expensify.cash
                     </Text>
                 </View>
-
-                {children}
+                <View style={[styles.loginFormContainer]}>
```

Comment:
`styles.loginFormContainer`

This style name needs to change since we're referring to `login` still but this doesn't have anything to do with the login form anymore.